src/lib/connectors/cdp/cdp-launcher.ts

```diff
@@ -31,7 +31,9 @@ export class CDPLauncher extends Launcher {
 
         chromeFlags.push('--no-default-browser-check');
         if (isCI) {
-            chromeFlags.push('--headless', '--disable-gpu');
+            // Running in `headless` mode is causing error in tests for `set-cookie` header.
+            // So the `--headless` flag was removed.
```

Comment:
@samccone do you have any idea why this happen? It looks like when we use the flag `--headless` the header `set-cookie` disappear in the event `onResponseReceive`, isn't it @qzhou1607?
Thanks!